include/mbgl/style/expression/number_format.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <mbgl/style/expression/expression.hpp>
+#include <mbgl/style/expression/parsing_context.hpp>
+
+namespace mbgl {
+namespace style {
+namespace expression {
+
+class NumberFormat final : public Expression {
+public:
+    NumberFormat(std::unique_ptr<Expression> number_,
+                 optional<std::unique_ptr<Expression>> locale_,
+                 optional<std::unique_ptr<Expression>> currency_,
+                 optional<std::unique_ptr<Expression>> minFractionDigits_,
+                 optional<std::unique_ptr<Expression>> maxFractionDigits_);
```

Comment:
complex class needs not-inlined destructor

Reply:
Done